src/backend/parser/Make.js

```diff
@@ -0,0 +1,26 @@
+var JisonLex = require('jison-lex');
+var Jison = require('jison');
+var fs = require('fs');
+var path = require('path');
+
+var dest = path.resolve(process.argv[2]); // first command-line argument
+
+try { // create directory if it does not exist
+    fs.mkdirSync(dest);
+} catch(e) {
+    if (e.code !== 'EEXIST' ) throw e;
```

Comment:
fs.existsSync is deprecated and that's why I haven't used it in the first place. I can add mkdirp npm package to do just that, I didn't want to install the whole package for a piece of code that is short and used only once.